search-commons/src/main/java/no/unit/nva/search2/common/Query.java

```diff
@@ -302,6 +303,14 @@ private Stream<Entry<K, QueryBuilder>> getQueryBuilders(K key) {
         } else if (opensearchQueryTools.isFuzzyKeywordKey(key)) {
             return new OpensearchQueryFuzzyKeyword<K>().buildQuery(key, value);
             // -> E M P T Y  S P A C E
+        } else if (opensearchQueryTools.isPublicFile(key)) {
+            return opensearchQueryTools.publishedFileQuery(key, value);
+            // -> E M P T Y  S P A C E
+        } else if (opensearchQueryTools.isOrganization(key)) {
+            return opensearchQueryTools.createSubunitsQuery(searchParameters, key);
+            // -> E M P T Y  S P A C E
+        } else if (opensearchQueryTools.isExcludeSubunits(key)) {
+            return null;
```

Comment:
if you don't do anything, you don't need this at all.

Reply:
It seems like this is needed, otherwise else condition will be executed and som 'default' filter/query created:

```
else {
            return new OpensearchQueryKeyword<K>().buildQuery(key, value);
        }
```